packages/code-du-travail-frontend/src/outils/CommonIndemniteDepart/steps/Anciennete/store/store.ts

```diff
@@ -148,7 +148,7 @@ const createAncienneteStore: StoreSlice<
                 : undefined,
           };
           const { result, ineligibility } = publicodes.calculate(situation);
-          if (result.value === 0 && ineligibility) {
+          if (!result?.value && ineligibility) {
```

Comment:
Le résultat peut maintenant être undefined au lieu de renvoyer une valeur 0 même si il n'y a pas d'éligibilité

Reply:
Ouais bof, je trouve ça moins bien perso....

Reply:
pourquoi on fait pas juste comme le fichier d'après ?
```suggestion
          if (ineligibility) {
```